templates/actors/character-sheet-2.hbs

```diff
@@ -148,8 +148,8 @@
                         <div class="top">
 
                             <div class="pips" data-prop="system.attributes.exhaustion">
-                                {{#each exhaustion}}
-                                {{#if (lt @index 3)}}
+                                {{#each exhaustion.pips}}
+                                {{#if (lt @index ../exhaustion.mid)}}
```

Comment:
Rather than doing this check in handlebars, maybe do something similar to what is done for the abilities on the new sheet and reduce them into a `left` and `right` array during data preparation. That might leave a bit more room for more tweaking of layout (when we get the ability to modify rendering context with Application V2).